<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChReagentUnLoadDlg</class>
 <widget class="BaseDlg" name="ChReagentUnLoadDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChReagentUnLoadDlg</string>
  </property>
  <widget class="QFrame" name="BottomBtnFrame">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>230</y>
     <width>596</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>596</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>596</width>
     <height>52</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
    <property name="spacing">
     <number>14</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>366</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="ok_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>138</y>
     <width>231</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>例如： 2,4,6-10</string>
   </property>
   <property name="colors" stdset="0">
    <string>disabled</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="select_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>138</y>
     <width>365</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>601</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>卸载指定试剂位</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
